mne/stats/regression.py

```diff
@@ -134,3 +136,193 @@ def _fit_lm(data, design_matrix, names):
+        appropriate - or using the decim keyword (see below).
+    events : np.ndarray of int, shape(n_events, 3)
+        An array where the first column corresponds to samples in raw
+        and the last to integer codes in event_id.
+    event_id : dict
+        As in Epochs; a dictionary where the values may be integers or
+        list-like collections of integers, corresponding to the 3rd column of
+        events, and the keys are condition names.
+    tmin : float | dict
+        If float, gives the lower limit (in seconds) for the time window for
+        which all event types' effects are estimated. If a dict, can be used to
+        specify time windows for specific event types: keys correspond to keys
+        in event_id and/or covariates; for missing values, the default (-.1) is
+        used.
+    tmax : float | dict
+        If float, gives the upper limit (in seconds) for the time window for
+        which all event types' effects are estimated. If a dict, can be used to
+        specify time windows for specific event types: keys correspond to keys
+        in event_id and/or covariates; for missing values, the default (1.) is
+        used.
+    covariates : dict-like | None
+        If dict-like, values have to be array-like and of the same length as
+        the columns in ```events```. Keys correspond to additional event
+        types/conditions to be estimated and are matched with the time points
+        given by the first column of ```events```. If None, only binary events
+        (from event_id) are used.
+    reject : None | dict
+        Activate rejection parameters based on peak-to-peak amplitude in
+        continuously selected subwindows. If reject is None, no rejection is
+        done. If dict, keys are types ('grad' | 'mag' | 'eeg' | 'eog' | 'ecg')
+        and values are the maximal peak-to-peak values to select rejected
+        epochs. E.g. reject = dict(grad=4000e-12, # T / m (gradiometers)
+                                   mag=4e-11, # T (magnetometers)
+                                   eeg=40e-5, # uV (EEG channels)
+                                   eog=250e-5 # uV (EOG channels))
+    tstep : float
+        Length of windows for peak-to-peak detection.
+    decim : int
+        Decimate by choosing only a subsample of data points. Highly
+        recommended for data recorded at high sampling frequencies, as
+        otherwise huge intermediate matrices have to be created and inverted.
+    picks : None | list
+        List of indices of channels to be included. If None, defaults to all
+        MEG and EEG channels.
+    solver : str | function
+        Either a function which takes as its inputs the predictor matrix X
+        and the observation matrix Y, and returns the coefficient matrix b;
+        or a string (for now, only 'pinv'), in which case the solver used
+        is dot(scipy.linalg.pinv(dot(X.T, X)), dot(X.T, Y.T)).T.
+
+    Returns
+    -------
+    ev_dict : dictionary
```

Comment:
I would say `evokeds`. Also say `dict` not `dictionary` as that's the convention we have everywhere else